src/renderer/components/AccountView.tsx

```diff
@@ -37,28 +38,64 @@ const explorerURL = (net: Net, address: string) => {
   }
 };
 
-function AccountView(props: { pubKey: string | undefined }) {
+function AccountView(props: { pubKey: string }) {
   const { pubKey } = props;
   const { net } = useAppSelector(selectValidatorNetworkState);
 
-  const [account, setSelectedAccountInfo] = useState<AccountInfo | undefined>(
-    undefined
-  );
+  if (!pubKey) {
+    pubKey = '12341234123412341234';
+  }
 
-  useInterval(() => {
+  const [account, setSelectedAccountInfo] = useState<AccountInfo>({
+    net,
+    pubKey,
+    accountId: new sol.PublicKey(pubKey),
+    accountInfo: sol.AccountInfo < Buffer > {},
```

Comment:
You want an object with the right properties (or change the type back to `AccountInfo | undefined` and make it `undefined`), you don't use the type when you specify the literal.

Reply:
example:

```
    accountInfo: {
      executable: false,
      lamports: 0,
      owner: new sol.PublicKey(''),
      data: Buffer.from(''),
    }
```